packages/communication-react/CHANGELOG.stable.md

```diff
@@ -1,9 +1,49 @@
 # Change Log - @azure/communication-react
 
-This log was last generated on Fri, 06 Oct 2023 19:59:45 GMT and should not be manually modified.
+This log was last generated on Mon, 06 Nov 2023 16:44:11 GMT and should not be manually modified.
 
 <!-- Start content -->
 
+## [1.10.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.10.0)
+
+Mon, 06 Nov 2023 16:44:11 GMT 
+[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.10.0-beta.1...@azure/communication-react_v1.10.0)
+
+You can now add your own custom buttons to the CallComposite and CallwithChatComposite on web (on desktop and mobile web). Custom Button Injection enables developers to add their own custom buttons to the call control bar component. Developers have the ability to add buttons to different sections of the call control bar: primary, secondary, or overflow areas of the control bar. With our simplified button API, you can now create a more immersive calling experience that integrates with your applications.
```

Comment:
Should have a sub-heading here for precedence like:
`### Custom Button Injection - General Availability`